charts/rekor/values.yaml

```diff
@@ -62,7 +62,7 @@ server:
     registry: gcr.io
     repository: projectsigstore/rekor-server
     pullPolicy: IfNotPresent
-    # crane digest gcr.io/projectsigstore/rekor-server:v0.12.0
+    # -- v0.12.0
```

Comment:
can you keep the `crane digest gcr.io/projectsigstore/rekor-server:v0.12.0` and maybe add in the others as well, thanks!

Reply:
I suggest a new PR to add a CONTRIBUTING.md.

I can give that a shot in a new PR.